src/Chunking.cpp

```diff
@@ -105,13 +215,28 @@ void Chunking::addEntityToChunks(std::set<Chunk*> chnks, const EntityRef ref) {
                 ((Mob*)other)->playersInView++;
         }
     }
+
+    if (ref.kind != EntityKind::PLAYER)
+        return; // nothing to send
+
+    if (!pcAppearances.empty())
+        sendAroundPacket(ref, pcAppearances, P_FE2CL_PC_AROUND);
+    if (!npcAppearances.empty())
+        sendAroundPacket(ref, npcAppearances, P_FE2CL_NPC_AROUND);
+    if (!shinyAppearances.empty())
+        sendAroundPacket(ref, shinyAppearances, P_FE2CL_SHINY_AROUND);
+    if (!transportationAppearances.empty())
+        sendAroundPacket(ref, transportationAppearances, P_FE2CL_TRANSPORTATION_AROUND);
 }
 
 void Chunking::removeEntityFromChunks(std::set<Chunk*> chnks, const EntityRef ref) {
     Entity *ent = ref.getEntity();
     bool alive = ent->isExtant();
 
-    // TODO: same as above
+    std::vector<Bucket<int32_t, MAX_IDS_PER_AROUND_DEL>> pcDisappearances;
+    std::vector<Bucket<int32_t, MAX_IDS_PER_AROUND_DEL>> npcDisappearances;
+    std::vector<Bucket<int32_t, MAX_IDS_PER_AROUND_DEL>> shinyDisappearances;
+    std::vector<Bucket<int32_t, MAX_IDS_PER_AROUND_DEL - 1>> transportationDisappearances;
```

Comment:
I understand that the packet has an extra member, but I don't like this magic gotcha being inline here. Please put a `MAX_TRANSPORTATION_IDS_PER_AROUND_DEL` constant up with the others, just for this one.